e2e-tests/portal/cypress/support/commands.js

```diff
@@ -31,6 +31,8 @@ import 'cypress-audit/commands';
 
 const { downloadFile } = require('./portal-api/fileshare');
 
+Cypress.Commands.add('reinsertMocks', require('./utils/reinsertMocks'));
```

Comment:
I believe this can be removed. are we using it?

Reply:
removed

Reply:
in PR I think it's still there. did you push your changes? :)